### PR TITLE
Config formatting

### PR DESCRIPTION
**Pull request by MagicLike**
I reformatted the configs for more consistency, removed an unneeded reboot instructions for the FPs and added the TWRP image to every config. (It needs to be updated - outdated version)

### CODE REVIEW

**Comment by MagicLike** [comment]
Any updates on this @tsterbak? Will this get merged before the 0.4.3 Release?

**Comment by tsterbak** [comment]
I want to go through the changes in the config and check them.
But it will be merged for 0.4.3-beta, which will be probably delayed to next week since there is no hurry now because so far there are no substantial bugs (as far as I can tell)

**Comment by MagicLike** [comment]
> I want to go through the changes in the config and check them. But it will be merged for 0.4.3-beta, which will be probably delayed to next week since there is no hurry now because so far there are no substantial bugs (as far as I can tell)

Alright, I will also check Samsung configs - I found one typo so far ^^

File: README.md [hunk 1]
```diff
@@ -217,13 +217,13 @@ Every step in the config file corresponds to one view in the application. These
...
   - `link_button_with_confirm`: Display a button that opens a browser with a given link, confirm afterwards. Link is given in `link`.
-- `content`: str; The content text displayed alongside the action of the step. Used to inform the user about whats going on.
-- `command`: [ONLY for call_button* steps] str; The command to run. One of `adb_reboot`, `adb_reboot_bootloader`, `adb_reboot_download`, `adb_sideload`, `adb_twrp_wipe_and_install`, `adb_twrp_copy_partitions`, `fastboot_boot_recovery`, `fastboot_unlock_with_code`, `fastboot_unlock`, `fastboot_oem_unlock`, `fastboot_get_unlock_data`, `fastboot_reboot`, `heimdall_flash_recovery`.
 - `img`: [OPTIONAL] Display an image on the left pane of the step view. Images are loaded from `openandroidinstaller/assets/imgs/`.
-- `allow_skip`: [OPTIONAL] boolean; If a skip button should be displayed to allow skipping this step. Can be useful when the bootloader is already unlocked.
```
Comment (tsterbak) [inline README.md:223]: Why is this removed? I would prefer to keep in it as documentation in case it is necessary.

Comment (MagicLike) [reply README.md:223]: I just moved it down to line 224 ^^

**Comment by MagicLike** [comment]
> This looks very good and cleans up the structure of the config files.
> 
> As soon as the duplicated `command` line for `fajita` is removed, this can be merged :)

Done! Everything should be ready for merge! :smiley:

**Review by tsterbak — APPROVED** [review]
